README.md

```diff
@@ -8,13 +8,19 @@ A command line interface is also included for convenience.
 
 ## Getting started
 
+<!--
 To install from PyPI:
 
+PIP EXTRA URL
+
     $ pip install speechmatics-python
```

Comment:
`pip install --extra-index-url https://test.pypi.org/legacy/ speechmatics-python` ?

Reply:
Removed that altogether (this is to be uncommented when this is finally published to PyPI)

Reply:
fair enough; btw - anything we want to do before we start publishing to pypi.org?

Reply:
Waiting for green light from Product

Reply:
@viren-nadkarni 👍